README.md

```diff
@@ -181,37 +181,66 @@ Just edit the issue title, description or comments as a regular buffer and use `
+| ---                          | ---              |
+| `OctoNvimDirty`              | `ErrorMsg`       |
+| `OctoNvimIssueTitle`         | `PreProc`        |
+| `OctoNvimIssueId`            | `Question`       |
+| `OctoNvimIssueOpen`          | `MoreMsg`        |
+| `OctoNvimIssueClosed`        | `ErrorMsg`       |
+| `OctoNvimEmpty`              | `Comment`        |
+| `OctoNvimFloat`              | `NormalNC`       |
+| `OctoNvimDate`               | `Comment`        |
+| `OctoNvimSymbol`             | `Comment`        |
+| `OctoNvimTimelineItemHeading`| `Comment`        |
+| `OctoNvimDetailsLabel`       | `Title`          |
+| `OctoNvimMissingDetails`     | `Comment`        |
+| `OctoNvimDetailsValue `      | `Identifier`     |
+| `OctoNvimDiffHunkPosition`   | `NormalFloat`    |
+| `OctoNvimCommentLine`        | `TabLineSel`     |
+| `OctoNvimEditable`           | `NormalFloat`    |
+| `OctoNvimViewer`             | GitHub color     |
+| `OctoNvimBubble`             | `NormalFloat`    |
+| `OctoNvimBubbleGreen`        | GitHub color     |  
+| `OctoNvimBubbleRed`          | GitHub color     |
+| `OctoNvimUser`               | `OctoNvimBubble` |
+| `OctoNvimUserViewer`         | `OctoNvimViewer` |
+| `OctoNvimReaction`           | `OctoNvimBubble` |
+| `OctoNvimReactionViewer`     | `OctoNvimViewer` |
+| `OctoNvimPassingTest`        | GitHub color     |
+| `OctoNvimFailingTest`        | GitHub color     |
+| `OctoNvimPullAdditions`      | GitHub color     |
+| `OctoNvimPullDeletions`      | GitHub color     |
+| `OctoNvimPullModifications`  | GitHub color     |
+
+The term `GitHub color` refers to the colors used in the WebUI.
+The (addition) `viewer` means the user of the plugin or more precisely the user authenticated via the `gh` CLI tool used to retrieve the data from GitHub.
 
 ## Settings
 
 - `g:octo_date_format`: Date format (default: "%Y %b %d %I:%M %p %Z")
 - `g:octo_remote_order`: Order to resolve the remote for the current working directory (default: ["upstream", "origin"])
 - `g:octo_qf_height`: Percent (when 0 < value < 1) or absolute (when value > 1) height of quickfix window (defaults to 20% relative)
+- `g:octo_bubble_delimiter_left`: Left (unicode) character to draw a bubble for labels etc. (default: "")
+- `g:octo_bubble_delimiter_right`: Right (unicode) character to draw a bubble for labels etc. (default: "")
+- `g:octo_icon_user`: Icon used to signal user names (default: "")
+- `g:octo_icon_reaction_viewer_hint`: Icon as alternative or to complement the highlighting of reactions by the viewer himself (default: "")
+
+## FAQ
+
+**How can I disable bubbles for XYZ?**
+
+Each text-object that makes use of a bubble (except labels) do use their own
+highlight group that linkes per default to the main bubble highlight group. To
+disable most bubbles at once you can simply link `OctoNvimBubble` to `Normal`.
+To only disable them for a certain plain do the same for the specific sub-group
+(e.g. `OctoNvimUser`)
```

Comment:
I had it like this before. But I kinda felt stupid to name everything that shows in a bubble with `OctoNvimBubble*`. Especially because we might use a bubble now, but eventually will change for some of them so a different design approach. It would be stupid to have to change these names, just we were too specific about them in the beginning. I think leaving this "detail" out in the name makes them not less readable/understandable, but would just restrict/hurt us on future changes. Do you agree?

Reply:
Ok, lets leave it AS IS for now. We should be good for merging! :tada: